packages/react-client/src/ReactFlightClient.js

```diff
@@ -648,7 +654,8 @@ export function reportGlobalError(response: Response, error: Error): void {
     }
   });
   if (enableProfilerTimer && enableComponentPerformanceTrack) {
-    flushComponentPerformance(getChunk(response, 0));
+    markAllTracksInOrder();
+    flushComponentPerformance(getChunk(response, 0), 0, -Infinity, -Infinity);
```

Comment:
I originally had that but it's not quite right because `getRoot()` is a public API and as such might end up with more stuff later on (like type checks of the arguments or something). However, it also means that the return type is constrained to a `Thenable` since that's what the public API is, but here I need it to be the chunk API.

Another consideration is that `getRoot()` being used in two places could also prevent it from being inlined in the wrapper unnecessarily.